universal-login-contracts/test/contracts/WalletMaster.js

```diff
@@ -54,7 +50,7 @@ describe('WalletMaster', async () => {
 
   describe('Signing', () => {
     it('initial key exist', async () => {
-      expect(await proxyAsWalletContract.keyExist(publicKey)).to.be.true;
+      expect(await proxyAsWalletContract.keyExist(wallet.address)).to.be.true;
```

Comment:
I would propose not to use wallet here as it might be confusing, instead use:
`const key = createKey();`
`const createKey = () => Wallet.createRandom().address;`
#createKey

Reply:
good point 👍